VotingApplication/VotingApplication.Web.Api.Tests/E2E/Helpers/Extensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using OpenQA.Selenium;
+using System;
+
+namespace VotingApplication.Web.Api.Tests.E2E.Helpers
+{
+    public static class Extensions
+    {
+        public static bool IsVisible(this IWebElement element)
+        {
+            try
+            {
+                return element.Displayed;
+            }
+            catch (NullReferenceException)
```

Comment:
It appears that IWebElements lazily load in values. If any one of them is actually evaluated, you will get a nullreferenceexception, presumably thrown by some mechanism in the underlying code.
My solution is a bit hacky, but it doesn't seem as though the library preovides a better alternative.